tool/teleport/common/teleport_test.go

```diff
@@ -173,6 +174,39 @@ func TestConfigure(t *testing.T) {
 	})
 }
 
+func TestDumpConfigFile(t *testing.T) {
+	tt := []struct {
+		name      string
+		outputURI string
+		contents  string
+		comment   string
+		shouldErr bool
```

Comment:
```suggestion
		assert require.ErrorAssertionFunc
```

Then in each test you can pass `require.Error` or `require.NoError`, and in the body of the test just call `tt.assert(t, err)`

Reply:
Awesome, thanks :+1: